### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by DcRonan**

### CODE REVIEW

**Comment by DcRonan** [comment]
@gurbuzertunga @lirad @dasileker @oracleot

File: index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+            </li>
+            <li class="top-icons" style="float: right;">
+                <a href="">
+                    <i class="fas fa-th fa-icon-styling"></i>
+                </a>
+            </li>
+        </ul>
+    </nav>
+    <main class="flex-container">
+        <iframe id="video" width="1001" height="562" src="https://www.youtube.com/embed/hu-q2zYwEYs" frameborder="0"
```
Comment (lirad) [inline index.html:44]: Try using 100% on the iframe for the video to always fill the container size

File: style.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    padding: 4px;
+    cursor: pointer;
+}
+
+.video-preview{
+    height: 100px;
+    width: 190px;   
+}
+
+.flex-container{
```
Comment (lirad) [inline style.css:46]: To become a flex container I suggest using the property: "display: flex"

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+             <li>
+                 <a href="">
+                     <img src="images/youtube.png" alt="home-logo">
+                 </a>
+             </li>
+             <li>
+                 <input type="search" width="20%">
+                 <button type="submit"><i class="fas fa-search"></i></button>
+             </li>
+             <li style="float: right;">
```
Comment (oracleot) [inline index.html]: I understand it's easier to keep certain styles in your html code but will suggest you keep all styling inside your css for easy management later

**Pull request closed without merging**